eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -270,28 +282,89 @@ instance EraPParams era => ToCBOR (GovActionsState era) where
 instance EraPParams era => FromCBOR (GovActionsState era) where
   fromCBOR = fromEraCBOR @era
 
+data HashProtected r a c
+  = Unprotected !a
+  | Protected !a !(PrevGovActionId r c)
```

Comment:
Currently spec assumes that there is always governance action id available, which is incorrect and will be fixed in the spec in the future
```suggestion
  | Protected !a !(StrictMaybe (PrevGovActionId r c))
```

Reply:
`Proteced` constructor is for the case where we have a previous govaction ID, while `Unprotected` is used when we don't have one.

Reply:
Maybe the naming is a bit confusing

Reply:
But it doesn't matter since I'm going to scrap that anyways